content/en/events/gsoc-2024.md

```diff
@@ -5,25 +5,53 @@ description = "Google Summer of Code 2024"
+For more information, see the GSoC website and/or reach out to the GSoC organizers.
 
-## Students
+Please only contact mentors about projects, not the program itself.
 
-Welcome to an incredible chance to dive into the vibrant Kubeflow community! Join us and become a valued contributor to our rapidly expanding network dedicated to AI/ML technologies on Kubernetes. Kubeflow mentors come with years of valuable experience and a passion for community growth and project enhancement, they're the guiding stars of our program. Immerse yourself in a dynamic environment where you'll not only gain insights from top industry experts but also sharpen your skills in open source collaboration, time management, and presentation prowess alongside making impactful technical contributions to projects of your interest. Our dedicated mentors will be by your side every step of the way, offering invaluable guidance and support throughout the project. Get ready to embark on an exciting journey of growth and learning!
+### Slack
 
-**If you need to reach out our mentors, join [Kubeflow Slack](https://www.kubeflow.org/docs/about/community/#kubeflow-slack) and look up your mentor or find project specific channel on Kubeflow slack**
+Please [__Join the Kubeflow Slack__](https://www.kubeflow.org/docs/about/community/#kubeflow-slack).
 
-## Project Ideas for the 2024
+Please __do not__ reach out privately to mentors, instead, start a thread in the `#gsoc-participants` channel so others can see the response. Be kind to our mentors, please search to see if your question has already been answered.
+
+### Meetings
+
+You may wish to attend the next community meeting for the group that is leading your chosen project, please see the calendar below for more information.
+
+<div id="calendar-container"></div>
+<script type="text/javascript">
+const timezone = Intl.DateTimeFormat().resolvedOptions().timeZone;
+const calender_src_list = [
+  // Kubeflow Community
+  "kubeflow.org_7l5vnbn8suj2se10sen81d9428%40group.calendar.google.com",
+  // KServe Community
+  "4fqdmu5fp4l0bgdlf4lm1atnsl2j4612%40import.calendar.google.com",
+];
+let calender_src = calender_src_list.map(src => `&src=${src}&color=%23A79B8E`).join('');
+const html = `<iframe src="https://calendar.google.com/calendar/embed?ctz=${timezone}&height=600&wkst=1&bgcolor=%23ffffff&showPrint=0&showDate=1&mode=AGENDA&showTitle=0${calender_src}" style="border:solid 1px #777" width="800" height="600" frameborder="0" scrolling="no"></iframe>`;
+document.getElementById('calendar-container').innerHTML = html;
+</script>
+
+## What if my proposal is not chosen?
```

Comment:
Should we section around contributing guides ?
For example:
Good starting point to learn how to contribute to Kubeflow is to follow [this guide](https://www.kubeflow.org/docs/about/contributing/).
Kubeflow components have individual contributing guide to understand the component better. For example:
- Katib [Contributing Guide](https://github.com/kubeflow/katib/blob/master/docs/developer-guide.md).
- Training Operator [Contributor Guide](https://github.com/kubeflow/training-operator/blob/master/docs/development/developer_guide.md).

Usually, issues with `good first issue` label is beginner friendly tasks that you can start to work on:

- Kubeflow Manifests [good first issues](https://github.com/kubeflow/manifests/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22).
- Training Operator [good first issues](https://github.com/kubeflow/training-operator/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22).
- Katib [good first issues](https://github.com/kubeflow/katib/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22).

Feel free to assign issues to yourself if you want to work on it, leave this comment in the issue `/assign @username`

Reply:
cc @juliusvonkohout for review

Reply:
This might be a little distracting on the page, perhaps we should just add a new section (with this info) to the top of the main contributor guide, and then link it here.

https://www.kubeflow.org/docs/about/contributing/